src/Sql/dbo/Tables/CollectionGroup.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[CollectionGroup] (
-    [CollectionId] UNIQUEIDENTIFIER NOT NULL,
-    [GroupId]      UNIQUEIDENTIFIER NOT NULL,
-    [ReadOnly]     BIT              NOT NULL,
+    [CollectionId]  UNIQUEIDENTIFIER NOT NULL,
+    [GroupId]       UNIQUEIDENTIFIER NOT NULL,
+    [ReadOnly]      BIT              NOT NULL,
+    [HidePasswords] BIT              NOT NULL DEFAULT 0,
```

Comment:
Can you please name the default constraint explicitly?

Reply:
I usually prefer not to use DEFAULT and handle instaitating it in the migration script.

Reply:
Resolved in the migration script instead.